assets/src/components/v2/normal_header.tsx

```diff
@@ -115,15 +115,23 @@ const NormalHeaderVersion: ComponentType<NormalHeaderVersionProps> = ({ version
   return <div className="normal-header-version">{version}</div>;
 };
 
+const NormalHeaderAccent = () => (
+  <div className="normal-header__accent-pattern-container">
+    <img className="normal-header__accent-pattern-image" src={imagePath(`dup-accent-pattern.svg`)} />
```

Comment:
To avoid referencing a specific app in a component shared by all, you could change `accentPattern?` to a string and pass in the `src` value. That will allow us to easily integrate different patterns into other apps if we ever need to.